### DISCUSSION TITLE
Hint

### DISCUSSION THREAD

**Discussion by cembora** [hbb20/CountryCodePickerProject#507] [Q&A]
Hi, I want to change all hints with "underscore" character. I can do it with replaceAll("[0-9]", "underscore") but this works only after first set. But if user change country "onCountrySelected" called before hint change(ccp.java line 851), so I can't capture new hint. 

Do you have a plan to add an option which is like "hintStyle='underscore'"?

(Some of our users didn't understand that hints and they said 'this is not my number' lol :)

**Comment by hbb20** [comment]
this is not my number😂😂

So basically you want to turn off hint right? How about setting this in XML?

app:ccp_hintExampleNumber="false"?

Did you try that?

**Reply by cembora** [reply]
Yes I made that, currently it is like that but it could be good if we can change the format I think. There can be "x","underscore","dash" or only one number formats.

Samples 
![image](https://user-images.githubusercontent.com/3588899/133034464-170c86c8-05a2-4399-a83d-660944a06e11.png)
![image](https://user-images.githubusercontent.com/3588899/133034531-535c7c48-c15f-4b37-a96e-f720fe88a7e2.png)
![image](https://user-images.githubusercontent.com/3588899/133034832-b12e4033-0f07-45a0-ad2c-cae7f3480414.png)

**Reply by hbb20** [reply]
Got it.. I am working on something similar in my personal project. Surprisingly there are more edge cases than I initially expected so it might take a while to get it right. Then I can explore how to integrate it with this.
